scripts/helpers/hrmp-helper.ts

```diff
@@ -121,11 +124,11 @@ export async function hrmpHelper(
           }
     );
     return xcmTransactorHrmpManageExtrinsic;
-  } catch (_) {
+  } else {
```

Comment:
This removes the catch functionality that was there previously. I think that was there in case someone was using this script on their own chain but didn't have the XCM transactor. You could have something like:

```javascript
try {
  if(!useTransactor) throw new Error("Default XCM send was forced by user.");
  // ... transactor code goes here ...
} 
catch(e) {
  console.log(`Not using XCM Transactor: {e.message ?? '[no message specified]'}`);
  // ... xcm message construction method here...
}
```

Reply:
Sure, can you push that fix?

Reply:
@albertov19 I've pushed the fix. You can review my change if you want